bcgov/mds
# MDS-3754: Preamble text in Draft permits.

# Main

-  Add ability to edit and save changes of preamble text in a permit amendment draft.

# Other
- Insert variables from the Variable Condition Menu during edit at the position of the cursor.

# How to test
- Open a NoW application.
- Go to a Draft tab and start a new Draft.
- New Preamble Text field is shown under Preamble section:
![image](https://user-images.githubusercontent.com/10526131/154566892-ca138a24-d10d-4ecb-8770-c6409be75326.png)
- Click on Edit Preamble Text to start editing the text and save changes. 
![image](https://user-images.githubusercontent.com/10526131/154573227-45bf1257-0019-4cf3-8776-c37c036d30c8.png)
- From de Condition Data Variables menu click on any variable to be inserted at the cursor position in the Preamble Text and Save
- Click on Download Draft to generate the document with changes in Preamble Text

# Notes
https://bcmines.atlassian.net/browse/MDS-3754
